webpack/assets/javascripts/react_app/redux/actions/toasts/index.js

```diff
@@ -0,0 +1,32 @@
+import {
+  TOASTS_ADD,
+  TOASTS_HIDE,
+  TOASTS_DELETE
+  // TOASTS_STORE_IN_LOCALSTORAGE,
+  // TOASTS_GET_FROM_LOCALSTORAGE
+} from '../../consts';
+import store from '../../../redux';
```

Comment:
this is usually not required, action creators return plain objects for connected components to dispatch , or thunk functions (dispatch => {} )